README.md

```diff
@@ -50,20 +55,25 @@ Kymatio requires:
 
 
 ### Standard installation (on CPU hardware)
-We strongly recommend running Kymatio in a Conda environment, because this simplifies the installation of PyTorch.
-One the aformentioned dependencies are installed, you may install the latest version of Kymatio by using the package manager `pip`, which will automatically download Kymatio from the Python Package Index (PyPI):
+We strongly recommend running Kymatio in an Anaconda environment, because this simplifies the installation of [PyTorch](https://pytorch.org). This is most easily achieved by running
+
+```
+conda install pytorch torchvision -c pytorch
+```
+
+Once PyTorch is installed, you may install the latest version of Kymatio using the package manager `pip`, which will automatically download Kymatio from the Python Package Index (PyPI):
 
 ```
 pip install kymatio
 ```
 
-Linux and macOS are the two operating systems that are officially supported by Kymatio.
+Linux and macOS are the two officially supported operating systems.
```

Comment:
MacOs?

Reply:
From what I understand, the approved capitalization is "macOS" (see https://en.wikipedia.org/wiki/MacOS#macOS).